### DISCUSSION TITLE
Focus Group: Workflow system, large scale workflows

### DISCUSSION THREAD

**Discussion by timosachsenberg** [OpenMS/OpenMS#7344] [General]
- [ ] current state of workflow systems, workflows
- [ ] Which platforms are currently used
- [ ] How to coordinate between different e.g., nf-core project, consolidate efforts
- [ ] How to get new developers

**Comment by timosachsenberg** [comment]
Will coordinate

**Comment by toanlhp** [comment]
Hello,
I'm running a workflow to analyze MS data from Agilent Q-TOF on KNIME. In the node IDScoreswitcher, I get this error message:
 **ERROR IDScoreSwitcher      3:11       Failing process stdout: []
ERROR IDScoreSwitcher      3:11       Failing process stderr: [Error: Unexpected internal error (Meta value '' not found for peptide hit with sequence 'HVGDLGNVTAGKDGVATVYIEDSVIALSGDHSIIGR', charge 4, score 6.25035e-15)]
ERROR IDScoreSwitcher      3:11       Return code: 8
ERROR IDScoreSwitcher      3:11       Execute failed: Failed to execute node IDScoreSwitcher**
Could anyone have idea how to solve this error? Thank you so much.

**Reply by jpfeuffer** [reply]
Hi, how is the node configured? Looks like you did not give a score name to switch to.

**Reply by jpfeuffer** [reply]
Let's discuss in #5307

**Comment by timosachsenberg** [comment]
Update: we had a meeting with Yasset, Hendrik, Julianus to discuss how to move forward with TMT workflow,s yesterday
Action items:
1. merge new matrix class
2. update IsobaricWorkflow branch, get it merged
3. Hendrik will implement MBR

**Comment by toanlhp** [comment]
Hi,
I found the way to solve it. Now I had a new problem that the result that after I ran Proteinquantifier node is 0 abundance in all groups. I'm running the workflow again and will update the result tomorrow for you. Thank  you.

**Comment by toanlhp** [comment]
> Let's discuss in #5307

OK

**Comment by timosachsenberg** [comment]
Report from dev meeting:
AP1: merge new matrix class (Timo, done)
AP2: update IsobaricWorkflow branch, get it merged (Julianus, Hendrik, WIP)
AP3: MBR (Hendrik, WIP)
AP4: some tool merger (PSMFeatureExtractor, Timo, WIP)

more?
